SecretOfManaClone.py

```diff
@@ -1,19 +1,123 @@
+        images.append(pygame.image.load("./art/dk_right_attack3.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_right_attack4.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_right_attack5.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_right_attack6.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_right_attack7.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_right_attack8.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_right_attack9.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_right_attack10.png").convert_alpha())
+    return images
+
+def attack_animation(direction, interval):
+    images = load_attack_animation(direction)
+    return images[interval]
+
+def load_walk_animation(direction):
+    images = []
+    if direction == "up":
+        images.append(pygame.image.load("./art/dk_up_walk1.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_up_walk2.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_up_walk3.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_up_walk4.png").convert_alpha())
+    
+    elif direction == "down":
+        images.append(pygame.image.load("./art/dk_down_walk1.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_walk2.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_walk3.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_down_walk4.png").convert_alpha())
+
+    elif direction == "right":
+        images.append(pygame.image.load("./art/dk_right_walk1.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_right_walk2.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_right_walk3.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_right_walk4.png").convert_alpha())
+
+    elif direction == "left":
+        images.append(pygame.image.load("./art/dk_left_walk1.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_left_walk2.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_left_walk3.png").convert_alpha())
+        images.append(pygame.image.load("./art/dk_left_walk4.png").convert_alpha())
+    return images
+
+def walk_animation(direction, interval):
+    images = load_walk_animation(direction)
+    return images[interval]
+
+def get_standing_direction(direction):
+    if direction == "up":
+        image = pygame.image.load("./art/dk_up.png").convert_alpha()
+    elif direction == "down":
+        image = pygame.image.load("./art/dk_down.png").convert_alpha()
+    elif direction == "right":
+        image = pygame.image.load("./art/dk_right.png").convert_alpha()
+    elif direction == "left":
+        image = pygame.image.load("./art/dk_left.png").convert_alpha()
```

Comment:
Ooh, I wonder if you could just use string interpolation to grab the image based on the direction?

```python
return pygame.image.load("./art/dk_%s.png" % direction).convert_alpha()
```